Stratis.SmartContracts/ISerializer.cs

```diff
@@ -91,6 +91,11 @@ public interface ISerializer
         /// </summary>
         ulong ToUInt64(byte[] val);
 
+        /// <summary>
+        /// Deserializes the first 32 bytes of a byte array into an uint256. If the given bytes are null, empty, or deserialization fails, returns default(uint256).
+        /// </summary>
+        UInt256 ToUInt256(byte[] val);
```

Comment:
Add Serialize method for UInt256 also add UInt128 methods for Serialize/Deserialize.